### PR TITLE
Add theme configuration

### LINKED ISSUE #161: Add theme configuration
verless should provide the option to add a theme configuration in a `theme.yml` file inside the particular theme directory. A possible structure of this configuration would be something like this:

```yaml
name: dark-theme # The theme name that users specify inside verless.yml
build:
    before:
        - sass style.scss generated/style.css
```

### PR DESCRIPTION
**Pull request by dominikbraun**
Fixes #161. Depends on #162.

### CODE REVIEW

**Review by aligator — COMMENTED** [review]
LGTM, but I added support for the "generated" directory and an example for sassc which uses that.

Could you please review the commit?

File: core/serve.go [hunk 1]
```diff
@@ -45,12 +46,14 @@ func Serve(path string, options ServeOptions) error {
 	done := make(chan bool)
 	rebuildCh := make(chan string)
 
+	fmt.Println(theme.GeneratedDir(filepath.Join(path, config.ThemesDir), cfg.Theme))
```
Comment (dominikbraun) [inline core/serve.go:49]: Is this intentional? :)

Comment (aligator) [reply core/serve.go:49]: no

Comment (dominikbraun) [reply core/serve.go:49]: Should I remove it or are going to do that yourself?

Comment (aligator) [reply core/serve.go:49]: I haven't much time till Monday, so you can do it.

Comment (aligator) [inline core/serve.go:49]:
```suggestion
```

Comment (aligator) [reply core/serve.go]: removed it

**Pull request merged**